apps/web-giddh/src/app/settings/permissions/form/form.component.ts

```diff
@@ -82,8 +89,8 @@ export class SettingPermissionFormComponent implements OnInit, OnDestroy {
             this.initAcForm();
         }
         // reset form
-        this.createPermissionInProcess$.pipe(takeUntil(this.destroyed$)).subscribe((val) => {
-            if (val) {
+        this.createPermissionSuccess$.pipe(takeUntil(this.destroyed$)).subscribe((value) => {
```

Comment:
Can we delete createPermissionInProcess$ if it is not used

Reply:
No! we can't bcz we have used it in component's template